[peer] 01:24:25.457185 honest.go:107: 3Importing modules...
../ML/code/utils.py:223: FutureWarning: Method .as_matrix will be removed in a future version. Use .values instead.
  data = data.as_matrix()
../ML/code/utils.py:94: FutureWarning: Method .as_matrix will be removed in a future version. Use .values instead.
  credit = df.ix[1:nn, 1:dd].as_matrix()
[peer] 01:24:32.195029 honest.go:122: 3Sucessfully pulled dataset. Features: 25
[peer] 01:24:32.195207 main.go:479: 3:Announcing myself to network
[peer] 01:24:32.195252 main.go:484: 3:Calling {198.162.52.126 %!s(int=8000) }
[peer] 01:24:32.195270 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.195281 main.go:507: 3Making RPC call
[peer] 01:24:32.195293 main.go:617: 3:Peer started. Receiving on 8003
[peer] 01:24:32.195448 main.go:513: 3RPC dial successful
[peer] 01:24:32.195463 main.go:516: 3:Calling RPC:198.162.52.126:8000
[peer] 01:24:32.195476 main.go:518: 3:RPC called198.162.52.126:8000
[peer] 01:24:32.196138 main.go:525: 3:Announced myself to a fellow peer at port. Got lastest chain
[peer] 01:24:32.196161 honest.go:340: Received chain length:4
[peer] 01:24:32.196170 honest.go:341: Appended chain length:4
[peer] 01:24:32.196177 main.go:543: 3Got lastest chain until iteration 2
[peer] 01:24:32.196218 main.go:484: 3:Calling {198.162.52.126 %!s(int=8001) }
[peer] 01:24:32.196229 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.196238 main.go:507: 3Making RPC call
[err] 01:24:32.196298 main.go:294: Peer offline.Couldn't connect to peer: 198.162.52.126:8001, err = dial tcp 198.162.52.126:8001: connect: connection refused
[peer] 01:24:32.196315 main.go:484: 3:Calling {198.162.52.126 %!s(int=8002) }
[peer] 01:24:32.196323 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.196330 main.go:507: 3Making RPC call
[err] 01:24:32.196369 main.go:294: Peer offline.Couldn't connect to peer: 198.162.52.126:8002, err = dial tcp 198.162.52.126:8002: connect: connection refused
[peer] 01:24:32.196383 main.go:484: 3:Calling {198.162.52.126 %!s(int=8004) }
[peer] 01:24:32.196390 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.196397 main.go:507: 3Making RPC call
[err] 01:24:32.196434 main.go:294: Peer offline.Couldn't connect to peer: 198.162.52.126:8004, err = dial tcp 198.162.52.126:8004: connect: connection refused
[peer] 01:24:32.196448 main.go:484: 3:Calling {198.162.52.126 %!s(int=8005) }
[peer] 01:24:32.196455 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.196462 main.go:507: 3Making RPC call
[err] 01:24:32.196498 main.go:294: Peer offline.Couldn't connect to peer: 198.162.52.126:8005, err = dial tcp 198.162.52.126:8005: connect: connection refused
[peer] 01:24:32.196512 main.go:484: 3:Calling {198.162.52.126 %!s(int=8006) }
[peer] 01:24:32.196519 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.196525 main.go:507: 3Making RPC call
[peer] 01:24:32.196580 main.go:513: 3RPC dial successful
[peer] 01:24:32.196592 main.go:516: 3:Calling RPC:198.162.52.126:8006
[peer] 01:24:32.196600 main.go:518: 3:RPC called198.162.52.126:8006
[peer] 01:24:32.204874 main.go:484: 3:Calling {198.162.52.126 %!s(int=8007) }
[peer] 01:24:32.204904 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.204915 main.go:507: 3Making RPC call
[err] 01:24:32.205002 main.go:294: Peer offline.Couldn't connect to peer: 198.162.52.126:8007, err = dial tcp 198.162.52.126:8007: connect: connection refused
[peer] 01:24:32.205019 main.go:484: 3:Calling {198.162.52.126 %!s(int=8008) }
[peer] 01:24:32.205025 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.205034 main.go:507: 3Making RPC call
[err] 01:24:32.205083 main.go:294: Peer offline.Couldn't connect to peer: 198.162.52.126:8008, err = dial tcp 198.162.52.126:8008: connect: connection refused
[peer] 01:24:32.205102 main.go:484: 3:Calling {198.162.52.126 %!s(int=8009) }
[peer] 01:24:32.205112 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.205118 main.go:507: 3Making RPC call
[err] 01:24:32.205164 main.go:294: Peer offline.Couldn't connect to peer: 198.162.52.126:8009, err = dial tcp 198.162.52.126:8009: connect: connection refused
[peer] 01:24:32.205191 main.go:484: 3:Calling {198.162.52.57 %!s(int=8010) }
[peer] 01:24:32.205202 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.205210 main.go:507: 3Making RPC call
[err] 01:24:32.205617 main.go:294: Peer offline.Couldn't connect to peer: 198.162.52.57:8010, err = dial tcp 198.162.52.57:8010: connect: connection refused
[peer] 01:24:32.205636 main.go:484: 3:Calling {198.162.52.57 %!s(int=8011) }
[peer] 01:24:32.205645 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.205653 main.go:507: 3Making RPC call
[peer] 01:24:32.206247 main.go:513: 3RPC dial successful
[peer] 01:24:32.206264 main.go:516: 3:Calling RPC:198.162.52.57:8011
[peer] 01:24:32.206273 main.go:518: 3:RPC called198.162.52.57:8011
[peer] 01:24:32.207643 main.go:525: 3:Announced myself to a fellow peer at port. Got lastest chain
[peer] 01:24:32.207684 main.go:484: 3:Calling {198.162.52.57 %!s(int=8012) }
[peer] 01:24:32.207694 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.207699 main.go:507: 3Making RPC call
[peer] 01:24:32.208001 main.go:513: 3RPC dial successful
[peer] 01:24:32.208014 main.go:516: 3:Calling RPC:198.162.52.57:8012
[peer] 01:24:32.208024 main.go:518: 3:RPC called198.162.52.57:8012
[peer] 01:24:32.209530 main.go:525: 3:Announced myself to a fellow peer at port. Got lastest chain
[peer] 01:24:32.209574 main.go:484: 3:Calling {198.162.52.57 %!s(int=8013) }
[peer] 01:24:32.209583 main.go:503: 3:Inside callRegisterRPC
[peer] 01:24:32.209588 main.go:507: 3Making RPC call
[peer] 01:24:32.209914 main.go:513: 3RPC dial successful
[peer] 01:24:32.209938 main.go:516: 3:Calling RPC:198.162.52.57:8013
[peer] 01:24:32.209947 main.go:518: 3:RPC called198.162.52.57:8013
[peer] 01:24:32.211329 main.go:525: 3:Announced myself to a fellow peer at port. Got lastest chain
[peer] 01:24:32.211349 honest.go:340: Received chain length:5
[peer] 01:24:32.211358 honest.go:341: Appended chain length:5
[peer] 01:24:32.211368 main.go:537: Iteration:2
[peer] 01:24:32.211378 main.go:538: Chain Length:5
[peer] 01:24:32.211387 main.go:540: Chain and iteration Count are inconsistent
